libs/platform/src/lib/components/button/button.component.ts

```diff
@@ -32,18 +32,24 @@ export class ButtonComponent extends BaseComponent {
     @Input()
     type: ButtonType;
```

Comment:
this need to be reviewed as the native button has `type` which is not exposed in the platform button. I would not mix the fundamental button type with the native.

Reply:
We should rename the property `buttonType`.

Reply:
As for adding the other attributes, I don’t think we should. The “platform” is an opinionated framework of components, we try to shield the end-user from complexity and provide a system of components that just work with each other.

The “core” library is designed for flexibility. It tries to add Fiori 3 UX design to components without getting in the way of the end-user. It allows the end-user to have more control over the configuration of components.

I think this is a key difference between the two libraries.

In my opinion, we should try be minimalistic in our approach of exposing optional native attributes. Until we see a need from our end-users or within our library, we should refrain from exposing additional attributes unless we think them essential.

Reply:
updated to buttonType.

Other attributes like formaction, former type,form,autofocus,form method,formnovalidate,form target will be added in feature PRs . Need to explore on them more.